packages/sdk/package.json

```diff
@@ -54,6 +54,7 @@
     "@waku/utils": "0.0.8",
     "@waku/relay": "0.0.3",
     "@waku/core": "0.0.20",
+    "@waku/interfaces": "0.0.15",
```

Comment:
There may be a `Protocol` enum in `@waku/interfaces` that generates JS code and hence justifies the presence in `dependencies`.